inc/classes/traits/trait-deprecated-class.php

```diff
@@ -0,0 +1,111 @@
+<?php
```

Comment:
@Tabrisrp `classes` will eventually go away. This trait should likely go into the `inc/deprecated` folder. What do you think?

Reply:
Good question, in a way it makes sense to have it in the `deprecated` folder since it's only used for deprecated classes. We will have to remember that the trait itself is not deprecated though. 

If it goes into the `inc/deprecated` folder, it will need to be autoloaded by the PSR-4 autoloader, so the filepath/filename needs to be changed to match the namespace/class name.

Reply:
@Screenfeed Let's do this:

- move this trait to `inc/deprecated`
- rename the file and trait to `DeprecatedClassTrait`
- change it's namespace to `WP_Rocket\deprecated`
- and then update where it's used.

Thanks!

Reply:
Done in ade1bb7.